lib/std/c/netbsd.zig

```diff
@@ -579,6 +579,12 @@ pub const PROT = struct {
     pub const READ = 1;
     pub const WRITE = 2;
     pub const EXEC = 4;
+    pub fn PROTECT(flag: u32) u32 {
+        return flag << 3;
+    }
+    pub fn PROTECT_EXTRACT(flag: u32) u32 {
+        return (flag << 3) & 0x7;
```

Comment:
the C code from the header you linked has a right shift, not a left shift:
```c
#define        PROT_MPROTECT_EXTRACT(x)        (((x) >> 3) & 0x7)
```

Reply:
true, thx.